examples/ola-recorder.cpp

```diff
@@ -164,16 +215,20 @@ int main(int argc, char *argv[]) {
                "Record a series of universes, or playback a previously "
                "recorded show.");
 
+  if (FLAGS_stop > 0 && FLAGS_stop < FLAGS_start) {
+    OLA_FATAL << "Stop time must be later than start time.";
```

Comment:
Did I ask before, what happens if stop==start, or do you just get that one frame played out?

Reply:
In the case of stop == start, only the final frame is used.  This is what the weirdness at the beginning of `ShowPlayer::SeekTo()` is supposed to handle.